src/ESPEasy.ino

```diff
@@ -307,9 +298,14 @@ void loop()
   if (wifiSetupConnect)
   {
     // try to connect for setup wizard
-    WifiConnect(1);
+    WiFiConnectRelaxed();
     wifiSetupConnect = false;
   }
+  if (wifiStatus != ESPEASY_WIFI_SERVICES_INITIALIZED) {
```

Comment:
These may be better suited for an enum. 

Switch statement would also be more appropriate and accommodate future logic changes.